ansible/roles/awscli/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Make sure that pip is installed
+  apt: name=python-pip state=installed
+  tags: 
+    - awscli
```

Comment:
Let's not put cruft in if it's not needed. Can you test if it's not needed and remove them if not? Ansible has changed since we started this repo, so we may have done it for the past behavior.